backend/common/src/main/java/ai/verta/modeldb/common/CommonDBUtil.java

```diff
@@ -310,6 +350,15 @@ protected void lockedMigration(String migrationName, RdbConfig rdbConfig)
   protected void runLiquibaseMigration(
       DatabaseConfig config, String liquibaseRootPath, ResourceAccessor resourceAccessor)
       throws InterruptedException, LiquibaseException, SQLException {
+    runLiquibaseMigration(config, liquibaseRootPath, resourceAccessor, Optional.empty());
+  }
```

Comment:
should we keep this around? seems like it's easy to forget to pass a very important argument

Reply:
UAC is still using this one, so we have to keep it for now.

Reply:
got it